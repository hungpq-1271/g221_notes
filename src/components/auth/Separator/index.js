import React from 'react'
import styled from 'styled-components'

const HeaderSeparation = styled.h4`
  width: 100%;
  text-align: center;
  border-bottom: 1px solid #818078;
  opacity: .5;
  line-height: 0.1em;
  margin: 10px 0 20px;
`
const SpanSeparation = styled.span`
  background: #fff;
  padding: 0 10px;
  font-family: Roboto;
  color: rgba(0, 0, 0, 0.87);
`

export default function Separator() {
  return (
    <HeaderSeparation>
      <SpanSeparation>OR</SpanSeparation>
    </HeaderSeparation>
  )
}
