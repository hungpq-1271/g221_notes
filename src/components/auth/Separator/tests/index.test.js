import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'

import Separator from '../index'

describe('<Separator />', () => {
  it('should render and match the snapshot', () => {
    const { container } = render(<Separator />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
