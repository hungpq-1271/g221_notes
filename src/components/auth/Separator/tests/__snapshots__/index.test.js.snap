// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> should render and match the snapshot 1`] = `
.c0 {
  width: 100%;
  text-align: center;
  border-bottom: 1px solid #818078;
  opacity: .5;
  line-height: 0.1em;
  margin: 10px 0 20px;
}

.c1 {
  background: #fff;
  padding: 0 10px;
  font-family: Roboto;
  color: rgba(0,0,0,0.87);
}

<h4
  class="c0"
>
  <span
    class="c1"
  >
    OR
  </span>
</h4>
`;
